Valhalla removed Modifier.IDENTITY

Failure link
------------

From [an internal build](https://hyc-runtimes-jenkins.swg-devops.com/job/Build_JDKnext_x86-64_linux_valhalla_Nightly/2124/consoleFull):

Optional info
-------------

Failure output (captured from console output)
---------------------------------------------
```
12:38:43  /home/jenkins/workspace/Build_JDKnext_x86-64_linux_valhalla_Nightly/build/linux-x86_64-server-release/support/j9jcl/java.base/share/classes/java/lang/Class.java:1788: error: cannot find symbol
12:38:43  				| Modifier.IDENTITY
12:38:43  				          ^
12:38:43    symbol:   variable IDENTITY
12:38:43    location: class Modifier
```

This was due to 
[8376430: [lworld] Remove Modifier.IDENTITY](https://github.com/openjdk/valhalla/commit/0f2f43b5cf38c165c16549c2286bafb8ce399a09)

FYI @hangshao0 @theresa-m

--- a/jcl/src/java.base/share/classes/java/lang/Class.java
+++ b/jcl/src/java.base/share/classes/java/lang/Class.java
@@ -2414,13 +2414,13 @@ public int getModifiers() {
 	if (isArray()) {
 		rawModifiers &= Modifier.PUBLIC | Modifier.PRIVATE | Modifier.PROTECTED
 /*[IF INLINE-TYPES]*/
-				| Modifier.IDENTITY
+				| AccessFlag.IDENTITY.mask()
 /*[ENDIF] INLINE-TYPES */
 				| Modifier.ABSTRACT | Modifier.FINAL;
 	} else {
 		rawModifiers &= Modifier.PUBLIC | Modifier.PRIVATE | Modifier.PROTECTED
 /*[IF INLINE-TYPES]*/
-				| Modifier.IDENTITY | Modifier.STRICT
+				| AccessFlag.IDENTITY.mask() | Modifier.STRICT
 /*[ENDIF] INLINE-TYPES */
 				| Modifier.STATIC | Modifier.FINAL | Modifier.INTERFACE
 				| Modifier.ABSTRACT | SYNTHETIC | ENUM | ANNOTATION;
@@ -3029,7 +3029,7 @@ public String toGenericString() {
 	 * Modifier.toString() is used later in this function which translates them to "synchronized" and "volatile",
 	 * which is incorrect. So remove these bits if they are set.
 	 */
-	modifiers &= ~(Modifier.IDENTITY | Modifier.STRICT);
+	modifiers &= ~(AccessFlag.IDENTITY.mask() | Modifier.STRICT);
 /*[ENDIF] INLINE-TYPES */
 
 	// Build generic string
@@ -6228,7 +6228,7 @@ public Class<?>[] getNestMembers()
 		Set<AccessFlag> flags = AccessFlag.maskToAccessFlags(maskedModifiers, location);
 /*[IF INLINE-TYPES]*/
 		if (isValhallaPreviewClassFile(getClassFileVersion())) {
-			if (isArrayClass && Modifier.isIdentity(rawModifiers)) {
+			if (isArrayClass && (rawModifiers & AccessFlag.IDENTITY.mask()) != 0) {
 				flags = new HashSet<>(flags);
 				flags.add(AccessFlag.IDENTITY);
 				flags = Collections.unmodifiableSet(flags);

